### PR TITLE
Practice with software development tools.

### PR DESCRIPTION
**Pull request by carlidev509**
- Add index.html.
- Link the HTML file with CSS file.
- Add a class called ".fav_color" to the level 1 heading in the HTML.
- Add main.css.
- Fix the broken MIT link in the README.md file.
- Fix the command example for the setup.
- Change "node-version" to 12.x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,58 +25,47 @@ After you're finished please remove all the comments and instructions!
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+<!-- > Add a link to your deployed project. -->
 
 - [Live Demo Link](https://google.com)
```
Comment (franclobo) [inline README.md:70]: - [ ] Unless ypu have deployed your app please 🙏🏽 replace this `[Live Demo Link](https://google.com)` with this `Coming soon...`

File: main.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.fav_color {
+  color: lightskyblue;
```
Comment (franclobo) [inline main.css:2]: - [ ] I think 🤔 the color should be blue 🟦, but the letters are black. Please correct the name of the color or use the [#rrggbb notation](https://www.w3.org/wiki/CSS/Properties/color/RGB)

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @carlidev509, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Correct Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] All the links in your README file should work as expected for professional documentation, please correct the MIT LICENSE link by replacing this `[MIT](./LICENSE)` with this `[MIT](./MIT.md)`, by correcting the path of the link with the correct name of the file it will return the correct information instead of 404 error.
- [ ] Please add a meaningful title to the description of your PR, we could edit the PR by pressing on the three dots in the right corner on the top 👇🏽 

image

- Kindly install the linter following the [instructions](https://github.com/microverseinc/linters-config/tree/master/html-css) step by step it will help you to have a clean code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @carlidev509 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966

## Highlights
✔No linter errors
✔Professional README
✔Good use of github workflow
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**